src/sass/claims-status.scss

```diff
@@ -288,51 +288,48 @@ p.first-of-type {
 
 .file-requirements {
   margin-bottom: 1.5em;
-  .file-requirement-header {
-    font-weight: bold;
-    margin-bottom: 0;
-  }
-  .file-requirement-text {
-    margin: 0;
-    &:last-child {
-      margin-bottom: 1em;
-    }
+}
+
+.file-requirement-header {
+  font-weight: bold;
+  margin-bottom: 0;
+}
+
+.file-requirement-text {
+  margin: 0;
+  &:last-child {
+    margin-bottom: 1em;
   }
 }
 
 .document-item-container {
   background-color: $color-gray-lightest;
   padding: 1em;
   margin-bottom: 1.5em;
-  .document-title-text-container {
-    padding-right: 1em;
-    word-break: break-all;
-    .document-size-text {
-      p {
-        margin: 0;
-      }
-    }
-  }
-  .remove-document-button {
-    > button {
-      margin: 0;
-      font-weight: bold;
-    }
-  }
   &:focus {
     outline: none;
   }
 }
 
+.document-title-text-container {
+  padding-right: 1em;
+  word-break: break-all;
+}
+
+.remove-document-button {
```

Comment:
Should we move this class to the button element? I don't see a reason for it to be on the containing div.

Reply:
Good point @jbalboni, change made.